pages/shared/data/tools.yaml

```diff
@@ -275,6 +275,17 @@ developer:
       - email
       - ads
       - stories
+  - name: React Storefront
+    official: true
```

Comment:
Could you please remove the `official` attribute here? That one is reserved for tools built by the AMP team. Thanks!

Reply:
@pbakaus ah my bad. didn't realize that. i've pushed a correction.